pkgs/servers/sql/postgresql/patches/prevent-output-cycle.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/src/port/Makefile b/src/port/Makefile
+index bfe1feb..1f9f0d3 100644
+--- a/src/port/Makefile
++++ b/src/port/Makefile
+@@ -152,9 +152,9 @@ pg_config_paths.h: $(top_builddir)/src/Makefile.global
+ 	echo "#define PGBINDIR \"$(bindir)\"" >$@
```

Comment:
For some reason `sharedir` and `bindir` in `$dev/bin/pg_config` point to `$dev`.